lesson06-01.html

```diff
@@ -0,0 +1,81 @@
+<html>
+  <head>
+    <meta charset="utf8" />
+    <style>
+      .active {
+        background-color: burlywood;
+      }
+    </style>
+  </head>
+  <body>
+      <ul id="related">
+        <li><span>apples</span><button>Добавить</button></li><hr>
+        <li><span>tangerines</span><button>Добавить</button></li><hr>
+        <li><span>pears</span><button>Добавить</button></li><hr>
+        <li><span>potatos</span><button>Добавить</button></li><hr>
+        <li><span>onions</span><button>Добавить</button></li><hr>
+
+      </ul>
+      <br>
+      <h4>Корзина</h4>
+      <ul id="list"></ul>
+      <script>
+        var products = [
+          {
+            name: 'apples',
+            kilogram: 1,
+            price: 100
+          },
+          {
+            name: 'tangerines',
+            kilogram: 1,
+            price: 120
+          },
+          {
+            name: 'pears',
+            kilogram: 1,
+            price: 130
+          },
+          {
+            name: 'potatos',
+            kilogram: 1,
+            price: 25
+          },
+          {
+            name: 'onions',
+            kilogram: 1,
+            price: 15
+          }
+        ];
+        var basket = [];
+        var $list = document.getElementById('list');
+        var $related = document.getElementById('related');
+
+        $related.addEventListener('click', handleAddClick);
+
+        function handleAddClick(event) {
+          if(event.target.tagName === 'BUTTON') {
+            var $span = event.target.previousSibling;
+
+            basket.push({ name: $span.textContent }, { kilogram: $span.textContent });
```

Comment:
Вот в этом месте нужно сделать проверку есть ли товар уже в корзине. Если нет, то положить с kilogram = 1
```suggestion
            basket.push({ name: $span.textContent, kilogram: 1 });
```